client/report_event_client.go

```diff
@@ -54,17 +52,21 @@ func NewReporterClient(kafkaProducer KafkaProducer, serviceName string) (*Report
 }
 
 // ReportError send an error report to the import-reporter
-func (c *ReporterClient) ReportError(instanceID string, eventContext string, err error, data log.Data) error {
-	log.ErrorC(eventContext, err, data)
-
+func (c ReporterClient) ReportError(instanceID string, errContext string, err error, data log.Data) error {
 	if len(instanceID) == 0 {
 		log.Info(instanceEmpty, nil)
-		return nil
+		return errors.New(instanceEmpty)
+	}
+	if len(errContext) == 0 {
+		log.Info(contextEmpty, nil)
+		return errors.New(contextEmpty)
 	}
 
+	log.ErrorC(errContext, err, data)
```

Comment:
Should this be an error? is there an error in this process, or is it just communicating about the error it is handling that came form another service? If it's just informing on what event it is handling, this should potentially be info level, as there's nothing wrong with this process, and we should expect an error level event to be logging in the originating service

Reply:
This is replacing the `log.Error(...)` call in the service. Its gets an error and it bubbles back up to the "top level" which calls the reporter. It logs out the error and then sends it.

Reply:
I am happy to move it out. Rely on the service to log its errors